spec/system/shared_examples/new_component_service.rb

```diff
@@ -1,7 +1,7 @@
 RSpec.shared_examples "new component page" do |component_name|
 
   before(:each) do
-    login_user
+    login_gds_user
```

Comment:
I would rather test for the actual role, instead of using the GDS user... same for all the below